sources/mep-kilogram-definition-en.adoc

```diff
@@ -192,73 +192,75 @@ Preserving the continuity of measurements traceable to an SI unit before and aft
 
 === Steps to ensure continuity
 
-Prior to the adoption of Resolution 1 of the 26^th^ CGPM (2018) <<cgpm-1-26>>, all mass standards used for the experimental determination of the Planck constant were calibrated by an "`extraordinary use`" of the IPK <<barat>>. Additionally, the BIPM ensemble of reference mass standards was calibrated.
+Prior to the adoption of Resolution 1 of the 26th CGPM (2018) <<cgpm-1-26>>, all mass standards used for the experimental determination of the Planck constant were calibrated by an "`extraordinary use`" of the IPK <<barat>>. Additionally, the BIPM ensemble of reference mass standards was calibrated.
```

Comment:
sources/mep-kelvin-definition-en.adoc used superscript ordinals, and this document uses plain ordinals. This is once again acquiescing to fatuous inconsistency. We should be forcing BIPM to pick one and stick to it.

Reply:
I've noticed this inconsistency earlier in BIPM SI Brochure, so Ronald checked with BIPM and we got the following answer (from https://github.com/metanorma/bipm-si-brochure/issues/16#issuecomment-699068021,  miswritten "subscript" instead of "superscript"):
> No, in BIPM publications we generally do not write the “th” as subscript. I can’t explain you why.

So I just removed these superscripts, which were also not present in the original document.

Reply:
Right, but you've actually added superscript ordinals in mep-kelvin-definition-en.adoc, so that is inconsistent.

"I can’t explain you why." I can explain why, it's because they've used different copies of Word with different autocorrect settings (quite possibly including different national settings), and they have never implemented any notion of editorial consistency. Like I said, let's not perpetuate that.

Reply:
I agree. Changed back these to plain ordinals.